web/components/SelectMultiple/index.tsx

```diff
@@ -0,0 +1,341 @@
+"use client";
+
+import {
+  autoUpdate,
+  flip,
+  offset,
+  shift,
+  size,
+  useDismiss,
+  useFloating,
+  useInteractions,
+} from "@floating-ui/react";
+import clsx from "clsx";
+import { useCallback, useEffect, useMemo, useState } from "react";
+import { FieldValues, Path, PathValue } from "react-hook-form";
+import { twMerge } from "tailwind-merge";
+import { DecoratedButton } from "../DecoratedButton";
+import { CaretIcon } from "../Icons/CaretIcon";
+import { CheckIcon } from "../Icons/CheckIcon";
+import { CloseIcon } from "../Icons/CloseIcon";
+import { TYPOGRAPHY, Typography } from "../Typography";
+
+type Item = { label: string; value: string };
+
+export type SelectMultipleProps<T extends FieldValues> = {
+  items: Item[];
+  values: PathValue<T, Path<T>>;
+  onChange: (value: string) => void;
+  onRemove: (value: string) => void;
+  label: string;
+  errors: any;
+  disabled: boolean;
+  required?: boolean;
+  className?: string;
+  helperText?: string;
+  selectAll?: () => void;
+};
+
+export const SelectMultiple = <T extends FieldValues>(
```

Comment:
@andy-t-wang 

New multiple select component: 

<details>
<summary>SelectMultiple</summary>


https://github.com/worldcoin/developer-portal/assets/89008845/4e1fb9d2-7f1e-441b-8861-d72f09d1cf3c

</details>

Reply:
Let me know if we need to adjust something from UI, for example maybe design of the selected items